### ISSUE TITLE
[BUG] Lua error

### ISSUE THREAD

**Issue by dapal-003** [teelolws/TLDRMissions#8]
Lua error: 
```
Message: Interface\AddOns\TLDRMissions\simulation.lua:1158: attempt to index field 'effects' (a nil value)
Time: Sat Apr  9 23:38:46 2022
Count: 1
Stack: Interface\AddOns\TLDRMissions\simulation.lua:1158: attempt to index field 'effects' (a nil value)
[string "=[C]"]: ?
[string "@Interface\AddOns\TLDRMissions\simulation.lua"]:1158: in function `setupPassives'
[string "@Interface\AddOns\TLDRMissions\simulation.lua"]:1419: in function <Interface\AddOns\TLDRMissions\simulation.lua:27>
[string "@Interface\AddOns\TLDRMissions\simulation.lua"]:1592: in function `Simulate'
[string "@Interface\AddOns\TLDRMissions\combinations.lua"]:7: in function `func'
[string "@Interface\AddOns\TLDRMissions\workhandler.lua"]:38: in function <Interface\AddOns\TLDRMissions\workhandler.lua:23>

Locals: 
```

from chat:
```
[23:38:46] onCooldown=false, 
[23:38:46] duration=0, 
[23:38:46] cooldown=4, 
[23:38:46] spellID=352
```
This lua error occurred on a kyrian mission table, please let me know if you need any additional information. 
Thank you for making this wonderful add-on

**Comment by teelolws** [comment]
Thanks. Its due to missing data about an Automa mission added in 9.2. The fix should be up when I'm done working on another part of the addon.

**Comment by teelolws** [comment]
Fixed in 9.2-009
